### DISCUSSION TITLE
FlorisBoard's Removal from Google Play

### DISCUSSION THREAD

**Discussion by gab12** [florisboard/florisboard#2667] [Q&A] [answered]
Hello,

Two years ago, FlorisBoard was available on Google Play, but it has since been removed  from the public store.
Do you know why?

It’s a bit of a shame, as even though I personally prefer F-Droid, Google Play provides unparalleled visibility for a project like this and could help many more people discover and use it.
I assume Patrickgold is already aware of this, so I don’t intend to submit a feature request, but I’m still curious to know the reason: is it an ethical choice, a lack of time, or something else?

Thank you for your response!

**Comment by lm41** [comment, marked as answer]
Hello,

Florisboard has not been removed from playstore. Florisboard will be released publicly in the playstore with v0.6. To install updates via the playstore you can follow the steps in the readme: 

**Stable**:
Join the [FlorisBoard Test Group](https://groups.google.com/g/florisboard-public-alpha-test), then visit the [testing page](https://play.google.com/apps/testing/dev.patrickgold.florisboard). Once joined and installed, updates will be delivered like for any other app. ([Store entry](https://play.google.com/store/apps/details?id=dev.patrickgold.florisboard))

**Preview**:
Join the [FlorisBoard Test Group](https://groups.google.com/g/florisboard-public-alpha-test), then visit the [preview testing page](https://play.google.com/apps/testing/dev.patrickgold.florisboard.beta). Once joined and installed, updates will be delivered like for any other app. ([Store entry](https://play.google.com/store/apps/details?id=dev.patrickgold.florisboard.beta))

I hope this helps :D

**Reply by hokindo9000** [reply]
yes
